canonical-web-and-design/snapcraft.io
# Add new hero on homepage

## Done

- Update hero on home page

## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1592

## QA

- Pull the branch
- Run the site using the dotrun snap with `$ dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [design](https://github.com/canonical-web-and-design/snapcraft.io/issues/2970#issuecomment-668460487)
- See that it works similar to the carousel on https://jaas.ai/
